### ISSUE TITLE
Fix tests failures in ast_test.py

### ISSUE THREAD

**Issue by razajafri** [NVIDIA/spark-rapids#11008]
These are the failures in ast_test.py
```
FAILED ../../../../integration_tests/src/main/python/ast_test.py::test_abs
FAILED ../../../../integration_tests/src/main/python/ast_test.py::test_addition
FAILED ../../../../integration_tests/src/main/python/ast_test.py::test_multi_tier_ast
FAILED ../../../../integration_tests/src/main/python/ast_test.py::test_multiplication
FAILED ../../../../integration_tests/src/main/python/ast_test.py::test_subtraction
FAILED ../../../../integration_tests/src/main/python/ast_test.py::test_unary_minus
```

**Comment by razajafri** [comment]
These will be all fixed by turning off ansi mode

**Comment by razajafri** [comment]
Closed by #11258
